src/actions/authActions.js

```diff
@@ -1,16 +1,26 @@
+import config from 'config'
+import appInsights from 'services/appInsightsService'
+
 export const LOGIN_IN_PROGRESS = 'LOGIN_IN_PROGRESS'
 export const USER_LOGGED_IN = 'USER_LOGGED_IN'
 export const USER_LOGGED_OUT = 'USER_LOGGED_OUT'
 export const USER_LOGIN_ERROR = 'USER_LOGIN_ERROR'
 
+
 export const loginInProgress = () => ({
   type: LOGIN_IN_PROGRESS
 })
 
-export const userLoggedIn = (user) => ({
-  type: USER_LOGGED_IN,
-  user
-})
+export const userLoggedIn = (user) =>{
+  if(config.useAppInsight){
```

Comment:
If we're going to make the use of application insights configurable (and we should), it might be best to check the condition only once in appinsightsService, that way we don't forget to put a guard clause around one of the calls to appinsights, and that way we don't need to import config everywhere we're importing appinsights. It might even be best to wrap the appinsights service in a service similar to AuthNService wrapping ADAL to allow the use of different telemetry platforms. That may be beyond the scope of this pull request, though.